add laravel upgrade link

kadirermantr opened the pull request:
With this development, a Laravel upgrade guide link has been added.

If the major version is less than 6, both the major and minor versions are taken into account together.

**Examples:**

- upgrade version = 5.7.15 --> guide link: https://laravel.com/docs/5.7/upgrade
- upgrade version = 8.5.23 --> guide link: https://laravel.com/docs/8.x/upgrade

semiherdogan commented:
thanks for the pr,
your code has some errors,
first you need to check if `this.upgradeVersion` is null or not before split.
second `"v5.5.22".split('.').slice(1, 3).join('.')` returns `'5.22'` but i think what you need is `5.5`.
except these two errors, i am ok with merging this feature. 
@dinncer do you have any comment on this?

kadirermantr commented:
> thanks for the pr, your code has some errors, first you need to check if `this.upgradeVersion` is null or not before split. second `"v5.5.22".split('.').slice(1, 3).join('.')` returns `'5.22'` but i think what you need is `5.5`. except these two errors, i am ok with merging this feature. @dinncer do you have any comment on this?

Yes, I noticed. To fix the problem I added an if condition and changed the version calculation methods.

semiherdogan commented:
i've updated code. @kadirermantr can you check if this approach is ok?
@dinncer any comment?

kadirermantr commented:
> i've updated code. @kadirermantr can you check if this approach is ok? @dinncer any comment?

It looks so good 🎉

dinncer commented:
@kadirermantr thank for the PR, @semiherdogan your update looks OK.
